formats/government.jsonnet

```diff
@@ -0,0 +1,25 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "government",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        start_date: {
```

Comment:
Could we call this `started_on` and the other one `ended_on` for consistency with similar naming in role appointments?

Reply:
Yeah, that sounds fine :+1:

Reply:
I've updated this Pull Request, and the govuk-content-schemas change now.